keyboards/dz60/keymaps/iso_br_abnt2/readme.md

```diff
@@ -0,0 +1,15 @@
+# ISO-BR abnt2 layout
+
+This is an ISO-BR abnt2 keymap for the DZ60 in a standard ISO layout.
+
+The top-left sends Escape normally, but becomes a quote key when
+tapped whilst either Fn or GUI are held down.
+
+## Layout Image
+
+![Layout Image](https://upload.wikimedia.org/wikipedia/commons/thumb/2/2c/KB_Portuguese.svg/1200px-KB_Portuguese.svg.png)
```

Comment:
This layout image doesn't match the physical layout (no split on the right Shift).